templates/address-space-controller/040-Service-address-space-controller.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: address-space-controller
+  labels:
+    app: enmasse
+spec:
+  ports:
+  - name: health
+    port: 8080
+    protocol: TCP
+    targetPort: http
+  selector:
+    component: address-space-controller
```

Comment:
Do we actually need a service or can we configure Prometheus to scrape the pod instead as done with the router and broker

Reply:
As discussed, service included to allow Prometheus to alert pods which are down